src/pages/Delivery/Delivery.layout.jsx

```diff
@@ -37,7 +38,7 @@ const Delivery = ({
   toChat,
 }) => {
   if (isLoading) {
-    return <pre>Loading...</pre>
+    return <LoadingView />
   }
 
   if (hasError) {
```

Comment:
What do you think about of remove this `if hasError`? Since we already have a catch for errors in delivery.index.jsx that is showing the modal and this view is not loaded if exists one error

https://github.com/Azordev/maps-project/blob/56e7513abe8c4176a85438e96418650dae9e0e93/src/pages/Delivery/index.jsx#L44-L52